osidb/filters.py

```diff
@@ -260,6 +260,33 @@ class FlawQLSchema(DjangoQLSchema):
     any field in the model, which is not desirable in this case.
     """
 
+    include = (
+        Flaw,
+        Affect,
+        Tracker,
+        Package,
+        FlawCVSS,
+        FlawAcknowledgment,
+        FlawReference,
+    )
+
+    suggest_options = {
+        Flaw: [
+            "impact",
+            "major_incident_state",
+            "nist_cvss_validation",
+            "requires_cve_description",
+            "source",
+            "workflow_state",
+            "components",
+            "owner",
+        ],
+        Affect: ["affectedness", "impact", "resolution", "ps_module", "ps_component"],
+        Tracker: ["resolution", "status", "type"],
+        FlawReference: ["type"],
+        FlawCVSS: ["issuer", "version"],
+    }
```

Comment:
Would it be possible to sort everything alphabetically?

Reply:
sorted

Reply:
Please look on items in `suggest_options` (i.e. `Affect`, `Flaw`, ...) and then on items in `Affect`.

Reply:
Third time's a charm